docs/LEARNING.md

```diff
@@ -1,13 +1,7 @@
 # Learning
 
-<!-- TODO: write document
+- The [Roc tutorial](https://www.roc-lang.org/tutorial) is a good place to start.
+- You can then go through the [code examples](https://www.roc-lang.org/examples).
+- Then you will want to get your hands dirty! Go through the Exercism practice exercises in this Roc track.
```

Comment:
This might be a personal thing, but the student is already on Exercism looking at Roc exercises so we don't need to remind them we have exercises. 

If we keep this line, it should be one sentence per line, and we currently have two sentences on this line.

Reply:
You're right, we can remove that line.

Reply:
@ageron I think the line is still there?

```suggestion
```

Reply:
Removed 👍